auth/email_action_links_test.go

```diff
@@ -309,6 +311,35 @@ func TestEmailVerificationLinkError(t *testing.T) {
 	}
 }
 
+func TestEmailVerificationSendEmail(t *testing.T) {
+	s := echoServer(testActionLinkResponse, t)
+	defer s.Close()
+
```

Comment:
Can we call this "TestEmailVerificationCustomOobLinkSettings" or similar?

Then, we can add 2 cases:
```
cases := []bool{true, false}

testActionCodeSettingsCustom := map[string]interface{}{}
// make a copy to avoid modifying the base maps.
for k, v := range testActionCodeSettings {
   testActionCodeSettingsCustom[k] = v 
}
testActionCodeSettingsMapCustom :=  map[string]interface{}{}
for k, v := range testActionCodeSettingsMapCustom {
   testActionCodeSettingsMapCustom[k] = v 
}

for _, returnOobLink := range cases {
  testActionCodeSettingsCustom.ReturnOobLink = returnOobLink
  testActionCodeSettingsCustomMap["returnOobLink"] = returnOobLink
 link, err := s.Client.EmailVerificationLink(..., testActionCodeSettingsCustom)
...
...
}
```

It would be great if you can add a similar test for EmailSignInLink and PasswordReset too. Thanks!